content/plugins/using-plugins.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: bt_wiki
+title: Working With Plugins
+category: Docs
+draft: false
+weight: 50
+
+---
+
+Cloudify Manager exposes a plugins API, used to allow the installation of Python dependencies from pre-packaged Python wheels. The plugins could come in handy for the following:
+
+* Allowing installation of Python dependencies without needing access to PyPi or the internet.
+* Eliminating the need for system level compilation tools such as gcc.
```

Comment:
so.. these two are simply not the point. The point is that you can use the plugins API to install Cloudify plugins.
The fact that they don't require post-compilation is irrelevant at the point and should be noted as a side note later on. Also, the internet thing is, once again, irrelevant. The fact that we needed a connection up until now was a fluke. Not needing access to the internet is not a feature.

The point was need to make here (including the first line) is:

Cloudify Manager exposes a plugins API used to allow users to upload Cloudify plugins to Cloudify Manager and install them.

Please add the notes regarding the no internet connection and no compilers needed to the end.

thanks